ReactiveCocoa/Swift/Flatten.swift

```diff
@@ -395,7 +395,7 @@ extension SignalProducerProtocol {
 	}
 	
 	/// `concat`s `self` onto initial `value`.
-	public func prefix(value: Value) -> SignalProducer<Value, Error> {
+	public func prefix(_ value: Value) -> SignalProducer<Value, Error> {
```

Comment:
This should retain the explicit `value:` label to distinguish it from the versions that take signals/producers. Same with `concat(value:)`.